crates/anvil/tests/it/sign.rs

```diff
@@ -276,30 +280,41 @@ async fn can_sign_typed_data_os() {
 
     assert_eq!(
       signature,
-      "0xedb0fa55ac67e3ca52b6bd6ee3576b193731adc2aff42151f67826932fa9f6191261ebdecc2c650204ff7625752b033293fb67ef5cfca78e16de359200040b761b".to_string()
+      "0xedb0fa55ac67e3ca52b6bd6ee3576b193731adc2aff42151f67826932fa9f6191261ebdecc2c650204ff7625752b033293fb67ef5cfca78e16de359200040b7601".to_string()
     );
 }
 
 #[tokio::test(flavor = "multi_thread")]
 async fn rejects_different_chain_id() {
     let (_api, handle) = spawn(NodeConfig::test()).await;
-    let provider = handle.ethers_http_provider();
+    let provider = ethers_http_provider(&handle.http_endpoint());
 
-    let wallet = handle.dev_wallets().next().unwrap();
+    let alloy_wallet = handle.dev_wallets().next().unwrap();
+    let wallet = Wallet::new_with_signer(
+        alloy_wallet.signer().clone(),
+        alloy_wallet.address().to_ethers(),
+        alloy_wallet.chain_id().unwrap(),
+    );
     let client = SignerMiddleware::new(provider, wallet.with_chain_id(Chain::Mainnet));
 
     let tx = TransactionRequest::new().to(Address::random()).value(100u64);
 
     let res = client.send_transaction(tx, None).await;
     let err = res.unwrap_err();
-    assert!(err.to_string().contains("signed for another chain"));
+    assert!(err.to_string().contains("signed for another chain"), "{}", err.to_string());
 }
 
 #[tokio::test(flavor = "multi_thread")]
 async fn rejects_invalid_chain_id() {
     let (_api, handle) = spawn(NodeConfig::test()).await;
-    let wallet = handle.dev_wallets().next().unwrap().with_chain_id(99u64);
-    let provider = handle.ethers_http_provider();
+    let alloy_wallet = handle.dev_wallets().next().unwrap();
+    let wallet = Wallet::new_with_signer(
+        alloy_wallet.signer().clone(),
+        alloy_wallet.address().to_ethers(),
+        alloy_wallet.chain_id().unwrap(),
+    );
```

Comment:
why not alloy wallet to ethers util?

Reply:
is this completed?

Reply:
done https://github.com/foundry-rs/foundry/pull/6808/commits/ede4871acb6815675e180b709f360b64ef446b84